packages/api/src/external/commonwell/organization.ts

```diff
@@ -71,6 +72,12 @@ export async function organizationToCommonwell(
 
 export const create = async (org: Organization): Promise<void> => {
   const { log, debug } = Util.out(`CW create - M oid ${org.oid}, id ${org.id}`);
+
+  if (!(await isCWDirectEnabledForCx(org.cxId))) {
+    console.log(`CW Direct is not enabled for org ${org.cxId}, skipping CW org creation`);
+    return;
```

Comment:
nit: I think there's a benefit on reading the code when we're explicit about what we're returning, like `return undefined;`

Happy to review with others if you guys think differently - could add a note on Notion.